templates/generic/digest.html

```diff
@@ -0,0 +1,89 @@
+{{period}} github digest ({{#topic}}{{.}}{{/topic}}{{^topic}}{{#repos}}{{shortname}}{{^last}}, {{/last}}{{/repos}}{{/topic}})
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Weekly digest</title>
```

Comment:
Maybe use the email subject for a more specific `<title>` (while I don't think email clients will show this, I can imagine reusing the template in a context targeting Web browsers)